py-bindings/generate_bindings.py

```diff
@@ -764,6 +764,11 @@ def filter_declarations(self):
         self.std_ns.class_('map<std::string, std::string >').include()
         self.std_ns.class_('map<std::string, std::string >').rename('mapStringToString')
         self.std_ns.class_('vector< ompl::PPM::Color >').rename('vectorPPMColor')
+        # Exclude the ProlateHyperspheroid Class which needs Eigen, and the associated member functions in the RNG
+        self.ompl_ns.class_('ProlateHyperspheroid').exclude()
+        self.ompl_ns.class_('RNG').member_functions('uniformProlateHyperspheroidSurface').exclude()
+        self.ompl_ns.class_('RNG').member_functions('uniformProlateHyperspheroid').exclude()
+
```

Comment:
I don't think there's an easy way to make Python bindings for Eigen, so I just excluded them. Correct if wrong, of course. :smile: